lib/modules/announcements.js

```diff
@@ -31,6 +31,8 @@ const recheckPostAfter = HOUR;
 
 const now = Date.now();
 
+const currentBrowser = BrowserDetect.browser;
```

Comment:
Thanks for the feedback. This was my initial thought as well, but my debugger was reporting the module as being out of scope when I tried this. Turns out my debugger was just being goofy because I wasn't reloading the plug-in from the Extensions menu after each build -___-

I'll clean this up.